music21/note.py

```diff
@@ -1784,6 +1785,49 @@ def pitchChanged(self):
         if self._chordAttached is not None:
             self._chordAttached.clearCache()
 
+    @property
+    def string(self):
```

Comment:
this new property and all properties below are rejected.  Note is the most common object in music21 and users who are not working with guitar music will never use these routines but will need to figure out what they mean in the docs.  (we don't have "accent" or "staccato" etc. either) -- and parallel routines would need to be added to Chord, etc.

In general for any mature open-source project, don't make additions or substantive changes to core parts of the system without a prior discussion.

Reply:
oops sorry about that, they were just for my personal convenience. Removed with https://github.com/cuthbertLab/music21/pull/1580/commits/0b2074f084def3497f79d66da4a351c59c387263